ministryofjustice/hmpps-developer-portal
# Improvements to the drift radiator

* Showing latest commit info alongside each commit

* Deriving drift and staleness from the latest commit rather than the dev version
    * This handles works better for components that use branch based deployments
    * Components within monorepos still use the dev version - as commits to the repo may mean no changes to an given service
    * Show diff based on latest main commit rather than dev version

* Flag services where we haven't received version updates for over a day
   * This is a bit of temporary thing whilst we fix the health ping

* Fix sorting so that situations where components where we can't calculate drift or staleness appear separate to those that are up to date

* Change the colour of services where we can't calculate drift/staleness to grey rather than green

* Update docs to include info about new calculation mechanism and behaviour around monorepos

* Removing a load of unused styling
